### ISSUE TITLE
Threading guide is unclear on reactor requirements

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#1793]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)| @exarkun reported|
|-|-|
|Trac ID|trac#1793|
|Type|defect|
|Created|2006-06-05 13:34:40Z|
|Branch|https://github.com/twisted/twisted/tree/threading-guide-reactor-1793|

http://twistedmatrix.com/projects/core/documentation/howto/threading.html includes examples that won't work unless you call "reactor.run()" but it does not talk about reactor.run() at all.

<details><summary>Searchable metadata</summary>

```
trac-id__1793 1793
type__defect defect
reporter__exarkun exarkun
priority__normal normal
milestone__ 
branch__branches_threading_guide_reactor_1793 branches/threading-guide-reactor-1793
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__documentation documentation
time__1149514480000000 1149514480000000
changetime__1218998065000000 1218998065000000
version__None None
owner__ 
cc__thijs
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_1"></a>|@thijstriemstra set owner to @thijstriemstra<br>@thijstriemstra set status to `assigned`|
|-|-|

So basically the samples need those missing reactor.run() statements?

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_2"></a>|@thijstriemstra commented|
|-|-|

(In [24563]) Branching to 'threading-guide-reactor-1793'

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)|@thijstriemstra removed owner<br>@thijstriemstra set status to `new`|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="washort's avatar" src="https://avatars.githubusercontent.com/u/947487?s=50" width="50" height="50">](https://github.com/washort)<a name="note_4"></a>|@washort set owner to @thijstriemstra|
|-|-|

Looks good. Please merge.

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_5"></a>|@thijstriemstra set status to `closed`|
|-|-|

(In [24614]) Merge threading-guide-reactor-1793: add missing reactor.run() call for some of the samples in the threading guide.

Author: thijs
Reviewer: washort
Fixes: [#1793](https://github.com/twisted/twisted/issues/1793)

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
